_episodes/05-preprocessor.md

```diff
@@ -0,0 +1,287 @@
+---
+title: "Working with preprocessors"
+teaching: 20?
+exercises: 15?
+questions:
+- "How do I set up a preprocessor?"
+- "Can I use different preprocessors for different variables?"
+- "Can I use different datasets for different variables?"
+objectives:
+- "Create a recipe with multiple preprocessors"
+- "Use different preprocessors for different variables"
+- "Run a recipe with variables from different datasets"
+keypoints:
+- "A recipe can run different preprocessors at the same time."
```

Comment:
Is it actually "running" different preprocessors at the same time? Maybe rephrasing to "A recipe can work with different preprocessors at the same time"?

Reply:
When recipes run in paralllel, do they only run the diagnostics in parallel or do preprocessor functions also go in parallel. At any rate, the change suggested by @hb326 is valid so will make it but good to clarify the other point on whether different preprocessors run at the same time.